sourmash/sbtmh.py

```diff
@@ -77,6 +80,207 @@ def data(self, new_data):
         self._data = new_data
 
 
+class LocalizedSBT(SBT):
+    """A Sequence Bloom Tree implementation which guarantees new leaves are plaaced in
+
+    The default node is a Bloom Filter (like the original implementation),
+    and the leaves are MinHash leaf class (in the sourmash.sbtmh.SigLeaf class)
+
+    Parameters
+    ----------
+    factory: Factory
+        Callable for generating new datastores for internal nodes.
+    d: int
+        Number of children for each internal node. Defaults to 2 (a binary tree)
+    storage: Storage, default: None
+        A Storage is any place where we can save and load data for the nodes.
+        If set to None, will use a FSStorage.
+
+    Notes
+    -----
+    We use two dicts to store the tree structure: One for the internal nodes,
+    and another for the leaves (datasets).
+    """
+
+    def __init__(self, factory, d=2, storage=None, track_abundance=False,
+                 do_containment=False):
+        if d != 2:
+            raise NotImplementedError("LocalizedSBT is only implemented for when the "
+                                      "number of children is 2, d=2")
+        SBT.__init__(self, factory=factory, d=d, storage=storage)
```

Comment:
I may not be understanding correctly, (not sure why you're passing in `self` here) but I think this would be easier to maintain if you called `super().__init__` here instead of `SBT.__init__` i think this is a more common pattern in python. But then again i could be misunderstanding the use case of the parent class 
```suggestion
        super().__init__(factory=factory, d=d, storage=storage)
```